extensions/find-permissions/main.ts

```diff
@@ -107,6 +107,14 @@ for (let i = 0; true; i++) {
 // Required sandboxing exceptions.
 const requiredPaths: string[] = [];
 
+// Abort if it doesn't work with all paths allowed.
+if (!test(["/"])) {
+  let msg =
```

Comment:
```suggestion
  const msg =
```
This is the suggestion from the deno checks failure.

Reply:
Interesting, I didn't find this when running it locally. But I guess as long as CI finds...